pkg/controller/manager/nad/controller.go

```diff
@@ -232,6 +260,8 @@ func constructJob(cur *batchv1.Job, namespace, image, dhcpServerAddr string, nad
 			ServiceAccountName: jobServiceAccountName,
 		},
 	}
+	backoffLimit := int32(2)
+	job.Spec.BackoffLimit = &backoffLimit
```

Comment:
should we consider cleaning up the pod if the job is succeeded to avoid data overwhelming?

Reply:
I suggest reserving the pods for debugging if there is something wrong.